README.md

```diff
@@ -4,6 +4,8 @@
 
 An SDK for Perpetual Protocol V2
```

Comment:
Since we already change the repo name to **sdk-curie**, is it better to change line 5 to  **  SDK curie for Perpetual Protocol V2** or something else?